packages/css/link.css

```diff
@@ -1,58 +1,56 @@
-  --dsc-link-color-hover: var(--ds-color-accent-text-default);
-  --dsc-link-color-active: var(--ds-color-accent-text-default);
-  --dsc-link-color-active-background: var(--ds-color-accent-surface-default);
-  --dsc-link-color-visited: var(--ds-global-purple-12);
-  --dsc-link-color-focus: var(--ds-color-focus-inner);
-  --dsc-link-color-focus-background: var(--ds-color-focus-outer);
+  --dsc-link-text-decoration-thickness--hover: max(2px, 0.125rem, 0.12em);
+  --dsc-link-text-decoration-thickness: max(1px, 0.0625rem);
 
-  position: relative;
   color: var(--dsc-link-color);
-  cursor: pointer;
-  text-decoration: underline;
-  text-decoration-thickness: max(1px, 0.0625rem);
+  outline: none;
+  text-decoration-style: solid;
+  text-decoration-thickness: var(--dsc-link-text-decoration-thickness);
   text-underline-offset: max(5px, 0.25rem);
-  display: inline-flex;
-  align-items: center;
-  gap: var(--ds-spacing-1);
-}
 
-.ds-link--accent {
-  --dsc-link-color: var(--ds-color-accent-text-subtle);
-  --dsc-link-color-hover: var(--ds-color-accent-text-default);
-  --dsc-link-color-active: var(--ds-color-accent-text-default);
-  --dsc-link-color-active-background: var(--ds-color-accent-surface-default);
-  --dsc-link-color-visited: var(--ds-global-purple-12);
-}
+  & svg {
+    vertical-align: middle; /* If adding SVG icon - align it to the text */
+  }
 
-.ds-link--neutral {
-  --dsc-link-color: var(--ds-color-neutral-text-default);
-  --dsc-link-color-hover: var(--ds-color-neutral-text-subtle);
-  --dsc-link-color-active: var(--ds-color-neutral-text-subtle);
-  --dsc-link-color-active-background: var(--ds-color-neutral-surface-default);
-  --dsc-link-color-visited: var(--ds-global-purple-12);
-}
+  /**
+    * Colors
+    */
+  &[data-ds-color="neutral"] {
+    --dsc-link-background--active: var(--ds-color-neutral-surface-default);
+    --dsc-link-color--active: var(--ds-color-neutral-text-subtle);
+    --dsc-link-color--hover: var(--ds-color-neutral-text-subtle);
+    --dsc-link-color--visited: var(--ds-global-purple-12);
+    --dsc-link-color: var(--ds-color-neutral-text-default);
+  }
 
-.ds-link:visited {
-  color: var(--dsc-link-color-visited);
-  text-decoration: none;
-}
+  /**
+    * States
+    */
+  &:visited {
+    color: var(--dsc-link-color--visited);
+  }
 
-.ds-link:hover {
-  color: var(--dsc-link-color-hover);
-  text-decoration-thickness: max(2px, 0.125rem, 0.12em);
-}
+  @media (hover: hover) and (pointer: fine) { /* Only use hover for non-touch devices to prevent sticky-hovering */
+    &:hover {
+      color: var(--dsc-link-color--hover);
+      text-decoration-thickness: var(--dsc-link-text-decoration-thickness--hover);
+    }
+  }
 
-.ds-link:focus-visible {
-  color: var(--dsc-link-color-focus);
-  background: var(--dsc-link-color-focus-background);
-  outline: none;
-}
+  &:focus-visible {
+    background: var(--dsc-link-background--focus);
+    color: var(--dsc-link-color--focus);
+  }
```

Comment:
Focus is wrong for our link.
Your CSS is this;
![image](https://github.com/user-attachments/assets/d76235af-c0cc-4444-9047-aaf914cde71a)

This is what we had before:
![image](https://github.com/user-attachments/assets/bb2474a7-9d04-4157-8400-d07944319eb0)